From ea230ba4915af68dc91e1798d7ce394984d64853 Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Fri, 2 May 2014 14:36:43 -0400
Subject: [misc] mei: fix function names in debug prints

Message-id: <1399041470-21330-11-git-send-email-prarit@redhat.com>
Patchwork-id: 79298
O-Subject: [RHEL7.1 PATCH BZ 1088953 10/77] mei: fix function names in debug prints
Bugzilla: 1088953
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Dean Nelson <dnelson@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1088953

commit 0da90747353c0f5b663c9c25fd56cd21440d222c
Author: Alexander Usyskin <alexander.usyskin@intel.com>
Date:   Mon Sep 16 23:44:42 2013 +0300

    mei: fix function names in debug prints

    Fix calling function names in debug prints.

    Signed-off-by: Alexander Usyskin <alexander.usyskin@intel.com>
    Signed-off-by: Tomas Winkler <tomas.winkler@intel.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/misc/mei/interrupt.c b/drivers/misc/mei/interrupt.c
index d27804e..be42c70 100644
--- a/drivers/misc/mei/interrupt.c
+++ b/drivers/misc/mei/interrupt.c
@@ -343,14 +343,14 @@ int mei_irq_read_handler(struct mei_device *dev,
 
  /* decide where to read the message too */
  if (!mei_hdr->host_addr) {
-  dev_dbg(&dev->pdev->dev, "call mei_irq_thread_read_bus_message.\n");
+  dev_dbg(&dev->pdev->dev, "call mei_hbm_dispatch.\n");
   mei_hbm_dispatch(dev, mei_hdr);
-  dev_dbg(&dev->pdev->dev, "end mei_irq_thread_read_bus_message.\n");
+  dev_dbg(&dev->pdev->dev, "end mei_hbm_dispatch.\n");
  } else if (mei_hdr->host_addr == dev->iamthif_cl.host_client_id &&
      (MEI_FILE_CONNECTED == dev->iamthif_cl.state) &&
      (dev->iamthif_state == MEI_IAMTHIF_READING)) {
 
-  dev_dbg(&dev->pdev->dev, "call mei_irq_thread_read_iamthif_message.\n");
+  dev_dbg(&dev->pdev->dev, "call mei_amthif_irq_read_msg.\n");
   dev_dbg(&dev->pdev->dev, MEI_HDR_FMT, MEI_HDR_PRM(mei_hdr));
 
   ret = mei_amthif_irq_read_msg(dev, mei_hdr, cmpl_list);
-- 
1.7.1